rosidl_generator_c/resource/idl__struct.h.em

```diff
@@ -60,10 +61,7 @@ from rosidl_parser.definition import Service
 }@
 @[for service in content.get_elements_of_type(Service)]@
 
-@{ hash_var = idl_structure_type_to_c_typename(service.namespaced_type) + '__TYPE_VERSION_HASH' }@
-// Note: this define is for MSVC, where the static const var can't be used in downstream aggregate initializers
-#define @(hash_var)__INIT @(type_hash_to_c_definition(type_hash['service'], line_final_backslash=True))
-static const rosidl_type_hash_t @(hash_var) = @(hash_var)__INIT;
+static const rosidl_type_hash_t @(idl_structure_type_to_c_typename(service.namespaced_type))__@(TYPE_HASH_VAR) = @(type_hash_to_c_definition(type_hash['service']));
```

Comment:
I am getting rid of the __INIT by using pointers in the other places this is used as an initializer. I think everything is nicer this way, but I wanted to point it out as a key change.